README.md

```diff
@@ -142,5 +176,41 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
 ```js
-// Your answer goes here...
+
+//IN JS File
+
+$(document).ready(() =>{
+  $(".post").on("click", () => {
+
+    let title = $(".title").val()
+    let body = $(".body").val()
+
+    $.ajax({
+      type: "POST",
+      data: {
+        post: {
+          title: title,
+          body: body
+        }
+      },
+      dataType: "json",
+      url: "/posts"
+    }).done((response) => {
+      console.log(response);
+    }).fail(() => {
+      console.log("Oh noes! Something went wrong!");
+    })
+  })
+})
+
+//In Viewfile
+  <label>Title:</label>
+  <input class="title" type="text">
+  <label>Content:</label>
+  <input class="body" type="text">
+
+
+
+//I'm pretty spotty on what I'm talking about here. I could be completely right or completely wrong and neither would really surprise me.
+
```

Comment:
you're close. you have a "post" selector in your js that'll trigger the `ajax` on click, so you need to create that in your html as well. 

you're ajax looks good tho!